### DISCUSSION TITLE
Status site design broken(nothing hardcoded)

### DISCUSSION THREAD

**Discussion by iqbalpb** [upptime/upptime#616] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/19872875/178655138-d72e3fb5-ad6b-4a12-a514-178c86608745.png)
Just forked and change yml file

**Comment by AnandChowdhary** [comment, marked as answer]
Can you please share a link to your site?

**Reply by iqbalpb** [reply]
Repo Link: https://github.com/iqbalpb01/xyz

**Reply by AnandChowdhary** [reply]
Looks like it works! :)
